rest/handler.go

```diff
@@ -301,7 +289,12 @@ func (h *handler) checkAuth(context *db.DatabaseContext) error {
 		return nil
 	}
 
-	defer checkAuthRollingMean.AddSince(time.Now())
+	// Record TotalAuthTime stat
+	defer func() {
+		// TODO: nanoseconds or milliseconds?
+		delta := time.Since(time.Now()).Nanoseconds()
```

Comment:
This is going to evaluate time.Now() when the defer runs.  Needs to be switched to something like:
```
defer func(t time.Time){
  ...time.Since(t)
}(time.Now())
```

Reply:
Thanks for catching that.  Fixed.